disco_aws_automation/disco_aws.py

```diff
@@ -193,10 +193,9 @@ def get_subnets(self, meta_network, hostclass):
         subnet_ips = self._get_hostclass_ip_address(hostclass)
 
         if not subnet_ips:
-            return meta_network.subnets
+            return [subnet.subnet for subnet in meta_network.subnets]
```

Comment:
I can see how it can be clearer. In this particular case (`subnet.subnet`), the first subnet is really a DiscoSubnet object, and the second one is an API-returned dictionary. I'll change it to `disco_subnet.aws_subnet` then.